### PR TITLE
[AAE-2605] - upgrade apiVersion and postgres chart to support Kubernetes version 1.16

### PR DESCRIPTION
**Pull request by SohelSaiyed**
[AAE-2605]

### CODE REVIEW

File: helm/alfresco-process-infrastructure/values.yaml [hunk 1]
```diff
@@ -82,11 +82,15 @@ persistence:
...
-    max_connections: 300
+  image:
+    repository: postgres
+    tag: "11.4"
+  postgresqlUsername: alfresco
+  postgresqlPassword: alfresco
+  postgresqlDatabase: postgres
+  postgresqlDataDir: /var/lib/postgresql/data/pgdata
+  postgresqlConfiguration:
+    max_connections: "1500"
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:93]: double quotes needed? 
out of curiosity, why did we change it from 300 to 1500?

File: helm/alfresco-process-infrastructure/values.yaml [hunk 2]
```diff
@@ -169,11 +173,15 @@ alfresco-modeling-app:
...
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:184]: see above comment

File: helm/alfresco-process-infrastructure/values.yaml (hunk 1 above)
Comment (mteodori) [reply helm/alfresco-process-infrastructure/values.yaml:93]: quotes are not needed almost everywhere

**Comment by ffazzini** [comment]
@SohelSaiyed are you removing the double quotes or not? Are we merging?

File: helm/alfresco-process-infrastructure/values.yaml (hunk 2 above)
Comment (SohelSaiyed) [reply helm/alfresco-process-infrastructure/values.yaml:184]: I will update and create new PR

**Pull request merged**